source/2019-07-26-the-ember-times-issue-108.md

```diff
@@ -23,12 +23,12 @@ learn how to use Sentry to track production errors 📈,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [How to Hire and Train Developers when You Don’t Use React 📚](https://medium.com/front-end-weekly/how-to-hire-and-train-developers-when-you-dont-use-react-42762e6b1a57)
   
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[@jenweber](https://github.com/jenweber) wrote an awesome blogpost on how to hire and train new developers.
+In 10 steps she points out the pain points of every new hire and how to solve them as a company.
```

Comment:
Hi, Anne. In general, with Markdown files, you will want to add a new line if a paragraph ends. Right now, you have one long paragraph. I think you had meant to have a few paragraphs instead?

If I understood you right, I think this sentence could be written as,

```suggestion
She provided 10 great tips on how you can help solve the pain points of hiring and training for your company.
```